server/schemas/project_public_setting.graphql

```diff
@@ -0,0 +1,43 @@
+type ProjectPublicSetting {
+  id: ID!
```

Comment:
[imo] Since it's just the settings we can recognize this as Value Object which doesn't have any id, I believe we can remove `id` field.